css/cart.css

```diff
@@ -0,0 +1,44 @@
+.cart-main {
+  max-width: 1200px;
+  margin: 1rem auto;
+}
+
+.cart-details {
+  flex-basis: 60%;
+}
+
+.price-details {
+  flex-basis: 40%;
+  height: fit-content;
+}
+
+.cart-action-btn {
+  display: flex;
+  justify-content: flex-end;
+  box-shadow: 0px 0px 4px var(--gray);
+  padding: 1rem 0.5rem;
+}
+
+.cart-action-btn .btn {
+  padding: 0.8rem 3rem;
+}
+
+.product-quantity-count {
+  width: 2.5rem;
+  height: 0.875rem;
+}
+
+.complementary-ouline-btn {
+  outline: 1px solid var(--complementary-color) !important;
```

Comment:
Is there a specific requirement to use `!important` here?

Reply:
Had to overwrite the CL styling, there is some problem with specificity related to the button in CL, this will be refactored after the changes.